opm/grid/polyhedralgrid/cartesianindexmapper.hh

```diff
@@ -73,6 +67,19 @@ namespace Dune
               coords[ 0 ] = gc ;
         }
 
+        // Only for unifying calls with CartesianIndexMapper<CpGrid> where levels are relevant.
+        int compressedLevelZeroSize() const
+        {
+            return grid_.size( 0 );
+        }
+        // Only for unifying calls with CartesianIndexMapper<CpGrid> where levels are relevant.
+        int cartesianIndexLevel( const int compressedElementIndex, const int level ) const
+        {
+            if (level) {
+                throw std::invalid_argument("Invalid level.\n");
+            }
+            return cartesianIndex(compressedElementIndex);
```

Comment:
This won't work as there migth be inactive cells in the parent LGR cells.

Reply:
We do not support LGRs in polyhedralGrid, do we? In principle, this addition is similar to cartesianCoordinateLevel() (line 84 same file) that was added back then, when supporting LGRs for CpGrid in opm-simulators.